packages/synthetix-main/contracts/storage/MarketPoolInfo.sol

```diff
@@ -0,0 +1,20 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title Stores information for specific actors in a Distribution.
+ */
+library MarketPoolInfo {
+    struct Data {
+        /**
+         * @dev . Needed to re-add the pool to the distribution when going back in range
+         */
+        uint128 liquidityAmount;
+        uint128 unused;
```

Comment:
Should we use a double underscore as nomenclature for all unused slots?

Reply:
if I use an underscore solidity complains that the lint is bad so im not sure.

can say for sure though that we can rename this stuff later